.github/workflows/stale.yml

```diff
@@ -12,11 +12,11 @@ jobs:
       - uses: actions/stale@v8
         id: stale
         with:
-          stale-issue-message: 'This issue has gone 30 days without any activity and meets the project’s definition of "stale". This will be auto-closed if there is no new activity over the next 30 days. If the issue is still relevant and active, you can simply comment with a "bump" to keep it open, or add the label "not_stale". Thanks for keeping our repository healthy!'
-          stale-pr-message: 'This PR has gone 30 days without any activity and meets the project’s definition of "stale". This will be auto-closed if there is no new activity over the next 30 days. If the issue is still relevant and active, you can simply comment with a "bump" to keep it open, or add the label "not_stale". Thanks for keeping our repository healthy!'
+          stale-issue-message: 'This issue has gone 30 days without any activity and meets the project’s definition of "stale". This will be auto-closed if there is no new activity over the next 7 days. If the issue is still relevant and active, you can simply comment with a "bump" to keep it open, or add the label "not_stale". Thanks for keeping our repository healthy!'
+          stale-pr-message: 'This PR has gone 30 days without any activity and meets the project’s definition of "stale". This will be auto-closed if there is no new activity over the next 7 days. If the issue is still relevant and active, you can simply comment with a "bump" to keep it open, or add the label "not_stale". Thanks for keeping our repository healthy!'
           stale-issue-label: 'stale'
           stale-pr-label: 'stale'
           days-before-stale: 30
-          days-before-close: 30
+          days-before-close: 7
           exempt-pr-labels: 'not_stale'
           exempt-issue-labels: 'not_stale'
```

Comment:
You also need to update the readme file https://github.com/mongodb/terraform-provider-mongodbatlas#autoclose-stale-issues-and-prs